CONTRIBUTING.md

```diff
@@ -55,6 +58,8 @@ ___
 
 
 ### Deployment
+TODO: Update this section
+
 Once built, Webpack outputs (e.g., index.html, JS and CSS files) are put into a tar archive, gzipped, and stored in
 Artifactory in the `maven-snapshot-local` repo. From there, deployments involve: a) pulling a particular artifact (referenced by git tag) out of Artifactory
 and b) copying the contents of the artifact to an S3 website bucket. For both staging and production deployments, these 
```

Comment:
@meganrm How much of this is still valid?

Reply:
it can all be deleted